synapse/lib/types.py

```diff
@@ -701,9 +699,35 @@ def __init__(self, modl, name, info, opts):
             'range=': self._storLiftRange,
         })
 
-    def norm(self, valu):
+        if modl.vers < (0, 2, 7):
```

Comment:
Maybe consider allowing types to cache this bool in their ctor and getting a callback when the model version is updated?